tests/phpunit/tests/option/networkOption.php

```diff
@@ -699,4 +699,97 @@ public function data_stored_as_empty_string() {
 			'null'         => array( null ),
 		);
 	}
+
+	/**
+	 * Tests that update_network_option() adds a non-existent option that uses a filtered default value.
+	 *
+	 * @ticket 59360
+	 *
+	 * @covers ::update_network_option
+	 */
+	public function test_update_network_option_should_add_option_with_filtered_default_value() {
+		global $wpdb;
+
+		$option        = 'update_network_option_custom_default';
+		$default_value = 'default-value';
+
+		add_filter(
+			"pre_site_option_{$option}",
```

Comment:
```suggestion
			"default_site_option_{$option}",
```
This one actually _should_ be `default_` because this specific test is filtering the default value